### PR TITLE
GPII-3042: Disabling HTTPS enforcement for preferences, flowmanager ingresses for dev clusters

### PR DESCRIPTION
**Pull request by natarajaya**

### CODE REVIEW

File: aws/modules/deploy/79-preferences-ingress.erb [hunk 1]
```diff
@@ -3,6 +3,10 @@ kind: Ingress
 metadata:
   name: preferences-ingress
   namespace: gpii
+  <% if ENV["TF_VAR_cluster_name"].start_with?("dev") %>
```
Comment (mrtyler) [inline aws/modules/deploy/79-preferences-ingress.erb]: Can you use `ENV["TF_VAR_cluster_name"].start_with?("prd.", "stg.")` instead, for consistency with other code? 

(Really, this should be a function `is_long_lived?` or something but fow now we can keep it down to one variant of this logic.)

File: aws/modules/deploy/89-flowmanager-ingress.erb [hunk 2]
```diff
@@ -3,6 +3,10 @@ kind: Ingress
 metadata:
   name: flowmanager-ingress
   namespace: gpii
+  <% if ENV["TF_VAR_cluster_name"].start_with?("dev") %>
```
Comment (mrtyler) [inline aws/modules/deploy/89-flowmanager-ingress.erb]: `ENV["TF_VAR_cluster_name"].start_with?("prd.", "stg.")`

File: aws/modules/deploy/79-preferences-ingress.erb (hunk 1 above)
Comment (natarajaya) [reply aws/modules/deploy/79-preferences-ingress.erb]: Done

File: aws/modules/deploy/89-flowmanager-ingress.erb (hunk 2 above)
Comment (natarajaya) [reply aws/modules/deploy/89-flowmanager-ingress.erb]: Done

**Comment by mrtyler** [comment]
LGTM

**Pull request merged**